### PR TITLE
Update Ember-CLI, fix deprecations

### PR DESCRIPTION
**Pull request by rwwagner90**
Removed ember-ajax and ember-data as dependencies as well, as I don't think they are needed, right?

### CODE REVIEW

**Comment by rwwagner90** [comment]
@j15e we can support 2.x and 1.13 with no deprecations by using `arguments` instead of passing actual params. Updated the PR to support both again.

File: LICENSE.md [hunk 1]
```diff
@@ -1,6 +1,6 @@
 The MIT License (MIT)
 
-Jean-Philippe Doyle, Copyright (c) 2015
+Copyright (c) 2016
```
Comment (j15e) [inline LICENSE.md]: I would like to keep my name on the copyright notice

Comment (rwwagner90) [inline LICENSE.md]: Didn't mean to remove it, was running `ember init` and thought the only change was the year.

**Comment by j15e** [comment]
Not using 1.13 anymore myself, but I do feel the pain for people upgrading huge apps (we did, took a lot of changes). Good thing if we can keep it compatible.

If you may restore the copyright notice I'll merge those changes & publish new version.

Out of curiosity, may I ask for what project you use this addon?

File: LICENSE.md (hunk 1 above)
Comment (j15e) [inline LICENSE.md]: Ah yeah, got it. Found it funny someone would deliberately remove my name from the copyright!

**Comment by rwwagner90** [comment]
I am using this for http://shipshape.io/ . Working on starting up an Ember consultancy and wanted to start improving my SEO. Check out my Ember addons and feel free to contribute 😄  https://github.com/shipshapecode

**Comment by rwwagner90** [comment]
@j15e good to go?

**Comment by j15e** [comment]
Yeah thanks

**Pull request merged**